lib/sqlalchemy/orm/session.py

```diff
@@ -1031,6 +1031,10 @@ def is_active(self) -> bool:
     def _is_transaction_boundary(self) -> bool:
         return self.nested or not self._parent
 
+    @property
+    def rollback_exception(self):
```

Comment:
let's keep this underscore, dont want to add more public accessors, note TransactionalContext uses underscored method names as its API